### PR TITLE
Add Salla / ecommerce

### PR DESCRIPTION
**Pull request by ahmedelq**
## Website:

https://salla.sa/

## Examples:
https://www.soollz.com/
https://elheen.com/
https://coffeesalla.com/
https://lakoalashop.com/

### CODE REVIEW

File: src/technologies/s.json [hunk 1]
```diff
@@ -16,6 +16,16 @@
     "scriptSrc": "\\.salesmanago\\.com/",
     "website": "https://www.salesmanago.com"
   },
+  "Salla": {
+	"cats": [6],
+	"description": "Salla is an ecommerce platform.",
+	"scriptSrc": "cdn\\.salla\\.sa/",
+	"headers": {
+	  "x-powered-by": "Salla"
```
Comment (nurbek91) [inline src/technologies/s.json:24]: Let's change the regex to avoid false positives
` "x-powered-by": "^Salla$"`

File: src/technologies/s.json [hunk 2]
```diff
@@ -16,6 +16,16 @@
...
+    },
+    "dom": "img[src*='cdn.salla.sa']",
```
Comment (nurbek91) [inline src/technologies/s.json:26]: Is this identifier necessary?
We need an example of sites where the header attribute and the scriptSrc are missing. Are there such cases?

**Review by nurbek91 — APPROVED** [review]
Hey mate, thanks for PR!!!
Need minor updates to make it perfect

File: src/technologies/s.json [hunk 3]
```diff
@@ -16,6 +16,16 @@
     "scriptSrc": "\\.salesmanago\\.com/",
     "website": "https://www.salesmanago.com"
   },
+  "Salla": {
```
[the remark this answers is not in the document]
Comment (ahmedelq) [reply src/technologies/s.json:19]: Yes this icon is correct.

File: src/technologies/s.json (hunk 2 above)
Comment (ahmedelq) [reply src/technologies/s.json:26]: Not aware about such cases. 
The `x-powered-by` is enough I guess.

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Okay, thanks for the detailed answers, waiting for updates from you.

**Comment by ahmedelq** [comment]
is there anything else you want me to add/remove?

**Comment by AliasIO** [comment]
All good, thanks!

**Pull request merged**